src/patternfly/upgrade-examples/AlertUpgradeExamples/examples/alert-upgrade-example-pf4-alert.hbs

```diff
@@ -0,0 +1,69 @@
+{{#> alert-upgrade-examples}}
+{{#> alert alert--modifier="pf-m-success" alert--attribute='aria-label="Success Notification"'}}
+  {{#> alert-icon alert-icon--success="true"}}
+  {{/alert-icon}}
+  {{#> alert-body}}
+    {{#> alert-title}}
+      {{#> accessibility accessibility--type="sr-only"}}Success: {{/accessibility}}
+        Great Job!
+    {{/alert-title}}
+    {{#> alert-description}}
+			This is really working out
+		{{/alert-description}}
+    {{#> alert-link alert-link--modifier="pf-c-button pf-m-link"}}
```

Comment:
This reads more like an `<a>`, like it will take you somewhere, than a `<button>`, which would perform some kind of action. Same for all of the links in the PF4 alert section.